doc/combined.json

```diff
@@ -0,0 +1 @@
+{"version": 1, "refs": {".zgroup": "{\"zarr_format\":2}", "foo/.zarray": "{\"chunks\":[4,5],\"compressor\":null,\"dtype\":\"<f8\",\"fill_value\":\"NaN\",\"filters\":null,\"order\":\"C\",\"shape\":[4,5],\"zarr_format\":2}", "foo/.zattrs": "{\"_ARRAY_DIMENSIONS\":[\"x\",\"y\"],\"coordinates\":\"z\"}", "foo/0.0": "base64:Dxw+e/RK6z/Iau/qGyXOPxjE5O6St8I/GnghejTg4j9GXx3xTV7iPwBbNsnGXq8/y7DgEsXk4j9U8rt0n2fPPw67F5DZydU/wj5q2OWC7z/VC9L/32ztP/C4P4XPVqM/Ry/m0M+R6z9cC0BpPB3oP0RRdt9y7tk/9GDp81P81T8w7i8oneDFP0YWAN0XQtk/rfwIfoeI5D+6TEUh7JLRPw==", "x/.zarray": "{\"chunks\":[4],\"compressor\":null,\"dtype\":\"<i8\",\"fill_value\":null,\"filters\":null,\"order\":\"C\",\"shape\":[4],\"zarr_format\":2}", "x/.zattrs": "{\"_ARRAY_DIMENSIONS\":[\"x\"]}", "x/0": "\n\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0014\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u001e\u0000\u0000\u0000\u0000\u0000\u0000\u0000(\u0000\u0000\u0000\u0000\u0000\u0000\u0000", "y/.zarray": "{\"chunks\":[5],\"compressor\":null,\"dtype\":\"<i8\",\"fill_value\":null,\"filters\":null,\"order\":\"C\",\"shape\":[5],\"zarr_format\":2}", "y/.zattrs": "{\"_ARRAY_DIMENSIONS\":[\"y\"],\"calendar\":\"proleptic_gregorian\",\"units\":\"days since 2000-01-01 00:00:00\"}", "y/0": "\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0002\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0003\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0004\u0000\u0000\u0000\u0000\u0000\u0000\u0000", "z/.zarray": "{\"chunks\":[4],\"compressor\":null,\"dtype\":\"|O\",\"fill_value\":null,\"filters\":[{\"allow_nan\":true,\"check_circular\":true,\"encoding\":\"utf-8\",\"ensure_ascii\":true,\"id\":\"json2\",\"indent\":null,\"separators\":[\",\",\":\"],\"skipkeys\":false,\"sort_keys\":true,\"strict\":true}],\"order\":\"C\",\"shape\":[4],\"zarr_format\":2}", "z/0": "[\"a\",\"b\",\"c\",\"d\",\"|O\",[4]]", "z/.zattrs": "{\"_ARRAY_DIMENSIONS\":[\"x\"]}"}}
```

Comment:
I don't see any references in here, only embedded data. What is this a reference set to?

Reply:
This is a reference to the `saved_on_disk.h5` generated inside the repo [here](https://docs.xarray.dev/en/stable/user-guide/io.html#hdf5)

Reply:
It would be good to mention that in the text; but all the data is inlined, so there are no references actually. "saved_on_disk.h5" does not appear in the JSON file.

You maybe wanted to reference to the HTTP address of that file, so that this would work whether the original file had been downloaded or not (which is more or less the point!).

Reply:
Oh, sorry: if this is generated, then I suppose there is no remote file you can refer to. In that case, you should write you are assuming the user has made this file.

Reply:
>  In that case, you should write you are assuming the user has made this file.

So I should not include the storage_options?

Reply:
If it is to be a local file you are referencing, then you don't need remote_* anything; but it could be useful to include the arguments anyway to demonstrate that they exist and in other uses of kerchunk are probably needed.